### ISSUE TITLE
Scraping breaks NES emulator

### ISSUE THREAD

**Issue by PeanutButterBoy** [Aloshi/EmulationStation#368]
I recently put together a Retropie (Raspberry Pi w/ Retropie SD card image). I loaded a few SNES and NES roms and used the built-in scraper to pull data for all of the roms; it pulled all the data fine. The problem is that none of the 20+ NES roms will load - I get a "failed to load rom" error every time. The SNES roms load perfectly fine. So here's what I did:

1. Deleted "gamelist.xml" in the "nes" folder and replaced it with a new (blank) "gamelist.xml"

2. Deleted all images that the scraper pulled (I'm not sure this was necessary after deleting gamelist.xml).

After doing this, all of the NES roms work fine! The problem is they don't have the scraped data, which was a nice addition. I searched but couldn't find this issue anywhere. Is this a known bug? Has anyone else experienced this?

**Comment by Aloshi** [comment]
This is really, really weird. The metadata is completely unconnected to how games are launched. I am guessing this is some sort of hardware problem - maybe insufficient power, or the Pi is running out of RAM from loading the images?

**Comment by PeanutButterBoy** [comment]
I agree. I'll look into it more. But I found another factor - all of my roms had a # symbol in the filename, and I found that even once I deleted the gamelist.xml, it still wouldn't load the game. I renamed the rom to remove # and it loaded fine. Should the # throw off rom loading?

I removed all the #, so I'm going to re-scrape all the roms and see if it's affected now. I'll report back.

**Comment by RichardvanHam** [comment]
hi it happens to me when i scraping roms no mater the emulator i tried when i hit about plus minus 400 roms  even whit a fresh install whit emulation 2.0 and 2.1 i think its a real bug for a 256mb pi b
